server/src/test/java/org/solarbank/server/unit/CalculateServiceTest.java

```diff
@@ -0,0 +1,35 @@
+package org.solarbank.server.unit;
```

Comment:
Our test folder structure should mimic the main folder structure. So in the case of this test it should be in `unit/service`.

Reply:
new directory added